db/migrate/20240310132236_add_email_notification_opt_in_to_organizations.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailNotificationOptInToOrganizations < ActiveRecord::Migration[7.0]
+  def change
+    add_column :organizations, :email_notification_opt_in, :boolean
```

Comment:
Please add `null: false, default: false` here. A null boolean doesn't make much sense since you then have three values, not two :smile: